packages/manager/modules/manager-pci-common/src/hooks/url/useParam.ts

```diff
@@ -0,0 +1,16 @@
+import { useParams } from 'react-router-dom';
+
+export const useParam = <T extends string>(
+  ...params: T[]
+): Record<T, string> => {
+  const urlParams = useParams();
+
+  return params.reduce((acc, param) => {
+    const value = urlParams[param];
+    if (!value) {
```

Comment:
what about params whose value equals 0 ?

Reply:
Yes you're right, I modified the assertion :)